front-end/src/app/components/device/device.component.html

```diff
@@ -1,32 +1,33 @@
 <div class="home-box device-box">
   <h3 class="header">Apparaten</h3>
   <div class="contents">
-    <table mat-table matSort #DeviceTableSort="matSort" [dataSource]="getDeviceList()" class="mat-elevation-z8 custom-md-table">
+    <table mat-table matSort #DeviceTableSort="matSort" [dataSource]="getDeviceList()"
+           class="mat-elevation-z8 custom-md-table">
       <ng-container matColumnDef="id">
-        <th mat-header-cell *matHeaderCellDef mat-sort-header class="custom-md-table-header"> Apparaat </th>
+        <th mat-header-cell *matHeaderCellDef mat-sort-header class="custom-md-table-header"> Apparaat</th>
```

Comment:
what happened with the spaces? either before and after or none - none is probably better

Reply:
no spaces anymore